CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 3.11.4 (2024-08-30)
+- Upgrade poolpeteer to fix contextPerRenderKey mode [#560](https://github.com/grafana/grafana-image-renderer/pull/560), [AgnesToulet](https://github.com/AgnesToulet)
```

Comment:
```suggestion
```
I'm not sure it should appear in the changelog as it fixes a bug introduced by the previous PR (a bug that was never released).